charts/bookkeeper/values.yaml

```diff
@@ -2,14 +2,10 @@ apiVersion: "bookkeeper.pravega.io/v1alpha1"
 kind: "BookkeeperCluster"
 metadata:
   name: "pravega-bk"
-
-spec:
```

Comment:
Why have we removed "spec:" from values file? Is this values.yaml tested?

Reply:
yes this file has been tested

Reply:
The "spec" had incorrectly been added to the values.yaml file before because of which values from this file were not being consumed in the pravega.yaml file. Hence I have corrected that here.